### ISSUE TITLE
Pip install -e Failed with error code 1 

### ISSUE THREAD

**Issue by Tsakunelson** [girder/large_image#332]
Hello all,
I followed the steps to install large_image on https://github.com/girder/large_image as standalone python module, But ends as failed with exit code 1 as on the screenshot. 
![image](https://user-images.githubusercontent.com/17685437/52015991-5bd4e100-24b1-11e9-88bc-21b4578f5fee.png)

Plus, when I go ahead and import the large_image module, it signals an import error and INFO:root notes:
![image](https://user-images.githubusercontent.com/17685437/52016095-ae160200-24b1-11e9-8b90-b8e057f91a37.png)

Has anyone faced this issue? Assistance most appreciated.

**Comment by manthey** [comment]
The first error is in installing the python libtiff module.  Do you have libtiff (the library) installed?  See the .travis.yml file for code on explicitly installing different versions of libtiff.  The `_multiarray_umath` error is from numpy -- it looks like something else is misinstalled.

**Comment by Tsakunelson** [comment]
Thanks for reaching out @manthey 
I figured there was a libtiff installed already so I had it uninstalled then successfully reinstalled large_image without the libtiff error anymore. Nonetheless, I still have three INFO:root notices when I import large_image ==>
![image](https://user-images.githubusercontent.com/17685437/52074546-1fa88b80-2558-11e9-9668-f12dc1cad7aa.png)

TiffileTileSource and MapnikTileSource still remain uninstalled. Is there a preferred way of installing them?

As for  "_multiarray_umath" is it a numpy version conflict? If so, what numpy version do I have to install

**Comment by manthey** [comment]
See the .travis.yml file for one way to install different versions of libtiff.

To use the mapnik tilesource, you need to install mapnik and GDAL and their python libraries.  Unless you are working with geospatial datasets, it probably isn't worth the effort.

We work numpy>1.10.2.

**Comment by Tsakunelson** [comment]
The error was solved after switching from numpy 1.15.2 to 1.10.2. Thanks @manthey 
Nonetheless, a Tilesource package not found error occurs when importing large_image ==> 

![image](https://user-images.githubusercontent.com/17685437/52097757-45568480-259a-11e9-9c83-0ae8b6e65b0d.png)

Whereas the package is found in the directory as on the following directory structure==>
```
large_image
 |
 | large_image #package
 | |
 | | __init__.py #class where error comes from
 | |
 | server #package
 | |
 | | tilesource #package
 | | cache_util #package

```

**Comment by manthey** [comment]
Since #329 has been merged, the layout of the repo has changed significantly, so this no longer should be an issue.
